.gitignore

```diff
@@ -14,3 +14,6 @@
 /.sass-cache
 Gemfile.lock
 .ruby-version
+
+.idea
```

Comment:
Of course there's a git global config for this. Silly me never thought to look. Removed.